content/en/tracing/connect_logs_and_traces/nodejs.md

```diff
@@ -29,9 +29,12 @@ const tracer = require('dd-trace').init({
 
 This enables automatic trace ID injection for `bunyan`, `paperplane`, `pino`, and `winston`.
 
+If you haven't done so already, we recommend configuring the NodeJS tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the smoothest
+experience for adding `env`, `service`, and `version` (see [Unified Service Tagging][1] for more details).
+
 **Note**: Automatic injection only works for logs formatted as JSON.
 
-## Manual Trace ID injection
+## Manual Injection into Logs
```

Comment:
Same here, the title sounds like it could be more explicit about what we are doing